quack_app/lib/screens/homepage/item_screen.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:auto_size_text/auto_size_text.dart';
+import 'package:flutter/material.dart';
+
+class ItemScreen extends StatelessWidget {
+  final String name;
+
+  const ItemScreen({Key? key, required this.name}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(title: const Text("")), // TODO: What should go here?
```

Comment:
I think the food name will work. Or you can put something generic, like "Detail"

Reply:
done